src/test/kotlin/no/nav/tilleggsstonader/sak/vilkår/stønadsperiode/StønadsperiodeServiceTest.kt

```diff
@@ -0,0 +1,240 @@
+package no.nav.tilleggsstonader.sak.vilkår.stønadsperiode
+
+import no.nav.tilleggsstonader.sak.IntegrationTest
+import no.nav.tilleggsstonader.sak.behandling.domain.BehandlingStatus
+import no.nav.tilleggsstonader.sak.util.BrukerContextUtil.testWithBrukerContext
+import no.nav.tilleggsstonader.sak.util.behandling
+import no.nav.tilleggsstonader.sak.vilkår.stønadsperiode.domain.StønadsperiodeRepository
+import no.nav.tilleggsstonader.sak.vilkår.stønadsperiode.dto.StønadsperiodeDto
+import no.nav.tilleggsstonader.sak.vilkår.stønadsperiode.dto.tilSortertDto
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.VilkårperiodeService
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.AktivitetType
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.MålgruppeType
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.SvarJaNei
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.dto.DelvilkårAktivitetDto
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.dto.DelvilkårMålgruppeDto
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.dto.OpprettVilkårperiode
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import java.time.LocalDate
+import java.util.UUID
+
+class StønadsperiodeServiceTest : IntegrationTest() {
+
+    @Autowired
+    lateinit var stønadsperiodeService: StønadsperiodeService
+
+    @Autowired
+    lateinit var stønadsperiodeRepository: StønadsperiodeRepository
+
+    @Autowired
+    lateinit var vilkårperiodeService: VilkårperiodeService
+
+    val SAKSHEH_A = "VL"
+    val SAKSHEH_B = "B"
```

Comment:
Skal det være `SAKSBEH`?

Reply:
Er ikke VL det vi ofte bruker for system? Burde noe annet brukes her (f.eks. A) for å ikke blande om målet ikke er at det skal være et system?